src/components/Form.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+const Formstyle = styled.form`
+  margin-top:1.8em;
+ input{
+   height:40px;
+   border-radius:var(--border-radius);
+   margin-right:20px;
+   width: 200px;
```

Comment:
This input should get
```suggestion
   width: 200px;
   border: 0;
   padding: 10px;
   font: inherit;
```
to make it look a little more like the design